src/confluent_kafka/schema_registry/protobuf.py

```diff
@@ -323,6 +323,8 @@ def __call__(self, message_type, ctx):
                 self._schema_id = self._registry.lookup_schema(
                     subject, self._schema).schema_id
 
+            self._known_subjects.add(subject)
```

Comment:
is this correct in the case auto register is not enabled and the schema isn't in schema registry?

Reply:
Hi @mhowlett,

I am not super familiar with the `auto_register:false` use case but I suspect that even if you don't auto register your schema you still want to avoid making a POST request on each [schema lookup](https://github.com/confluentinc/confluent-kafka-python/blob/d9a91a3d74c6447d53ba1c5733b0f99c2db2e259/src/confluent_kafka/schema_registry/schema_registry_client.py#L412-L414)?

The change suggested seems to be consistent with the [avro behaviour](https://github.com/confluentinc/confluent-kafka-python/blob/master/src/confluent_kafka/schema_registry/avro.py#L219-L223).

Cheers,
Matthieu